Form-Controls/index.html

```diff
@@ -13,6 +13,37 @@ <h1>Product Pick</h1>
     </header>
     <main>
       <form>
+        <div>
+          <label for="name">Name</label>
+          <input type="name" name="name" id="name" minlength="2" required />
+        </div>
+
+        <div>
+          <label for="email">E-mail address</label>
+          <input type="email" name="email" id="email" />
```

Comment:
This input has the right type here for an `email` - but if a user tries to submit the form without this field filled out, the validation doesn't stop the form from being submitted. Do we also need to declare something else on the input element here?